acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -88,11 +88,23 @@ def run_pcp_broker(host, instance=0)
 
 def kill_pcp_broker(host)
   on(host, "ps -C java -f | grep pcp-broker | sed 's/[^0-9]*//' | cut -d\\  -f1") do |result|
-    pid = result.stdout.chomp
-    on(host, "kill -9 #{pid}")
+    pids = result.stdout.chomp.split("\n")
```

Comment:
Minor comment: This method might now need renamed "kill_pcp_brokers" and/or have an optional argument to select which brokers to kill